### PR TITLE
Update downloading-workflow-artifacts.md

### PR DESCRIPTION
**Pull request by TheNH813**
Artifacts cannot be downloaded unless a user is logged in, and this is not specified in the current version of the page. Updating for clarity.

### Why:

It was unclear why I was not able to download artifacts for a project I wanted to help test. After doing some research, I concluded that even for public repositories, users must be logged in to access artifacts. Even for public repositories, the downloads links are not there unless a user is logged in. This information should be in the relevant documentation page so it is clear to users that this is a requirement.

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Added statement that artifacts cannot be downloaded unless a user is logged in.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@TheNH813 Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by TheNH813 — COMMENTED** [review]
@lecoursen
Made the requested changes to suggestion.

File: content/actions/managing-workflow-runs/downloading-workflow-artifacts.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Downloading workflow artifacts
 intro: You can download archived artifacts before they automatically expire.
+permissions: Authorized accounts with read access to a repository can download artifacts.
```
Comment (lecoursen) [inline content/actions/managing-workflow-runs/downloading-workflow-artifacts.md:4]: Thanks so much for making that change @TheNH813! I'm going to tweak it just slightly to really reinforce the fact that you need to be signed in as you so helpfully pointed out. 💖 

```suggestion
permissions: People who are signed into {% data variables.product.product_name %} and have read access to a repository can download workflow artifacts.
```

**Review by lecoursen — APPROVED** [review]

**Review by violethaze74 — COMMENTED** [review]
files

**Pull request merged**